model/src/main/kotlin/utils/DependencyGraphBuilder.kt

```diff
@@ -335,6 +335,12 @@ class DependencyGraphBuilder<D>(
         val dependencies2 = dependencies.associateBy { dependencyHandler.identifierFor(it) }
         if (!dependencies2.keys.containsAll(dependencies1)) return false
 
+        if (dependencyHandler::class.simpleName == "GradleDependencyHandler") {
```

Comment:
Some nits only: In the commit message, we prefer to start capitalized after the ":" in the title. Also the body's sentence should end with a dot.

Reply:
Also, should this be `dependencyHandler.javaClass.simpleName` instead?